rust/core-lib/assets/defaults.toml

```diff
@@ -27,6 +27,10 @@ word_wrap = false
 
 autodetect_whitespace = true
 
+syntax_files = "~/.config/xi/syntaxes"
+
+syntaxes_dump_path = "~/.config/xi/dumps/syntax.dump"
```

Comment:
we can't assume this path is in use; the frontend can provide an arbitrary config path (for instance on xi-mac the config path is `~/Library/Application Support/XiEditor`.

You should be able to copy how we handle the themes directory; if the user has provided a config directory, we just append the path to that.

Reply:
Okay, shall look into that.